tests/e2e/spec/adblocker.spec.js

```diff
@@ -0,0 +1,244 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+import { browser, expect } from '@wdio/globals';
+import {
+  enableExtension,
+  setPrivacyToggle,
+  setCustomFilters,
+} from '../utils.js';
+
+import { PAGE_DOMAIN, PAGE_URL as baseUrl } from '../wdio.conf.js';
+
+const PAGE_URL = baseUrl + 'adblocker/index.html';
```

Comment:
Change this to `ADBLOCKER_PAGE_URL` and use `PAGE_URL` as it was. It is easier to find usage of that in the code.